audio_switch.c

```diff
@@ -37,6 +37,7 @@ void showUsage(const char * appName) {
            "  -f format      : output format (cli/human/json). Defaults to human.\n"
            "  -t type        : device type (input/output/system/all).  Defaults to output.\n"
            "  -m mute        : sets the mute status (mute/unmute/toggle).  For input/output only.\n"
+           "  -A all          : Can be used with -m mute option above to set mute status for all devices, can't be used if toggle mute status is specified\n"
```

Comment:
There is no parameter `<all>` to this, right? Would `mute-all` and `unmute-all` to `-m` be better?

Reply:
@zuckschwerdt updated to use mute-all and unmute-all with -M and -U flags respectively

Reply:
The new `-M` and `-U` again don't need a parameter. It should e.g. be 
`" -M          : sets the mute status to mute for all devices of specified device type\n"`
and **not**
`" -M mute-all : sets the mute status to mute for all devices of specified device type\n"`

But my suggestion -- just a suggestion really -- was to use:
`"  -m mute        : sets the mute status (mute/mute-all/unmute/unmute-all/toggle).  For input/output only.\n"`
So we don't add new options.
But that is just my preference, your *modified* original is also a valid option (but could be confusing as you state in the text on when it applies)
`"  -A             : Can be used with -m mute option above to set mute status for all devices, can't be used if toggle mute status is specified\n"`

Reply:
I see now what you meant, updated it